src/core/server/graph/loaders/Comments.ts

```diff
@@ -384,6 +371,25 @@ export default (ctx: GraphContext) => ({
       story.isArchived
     ).then(primeCommentsFromConnection(ctx));
   },
+  allChildComments: async (
+    comment: Comment,
+    { first, orderBy }: CommentToRepliesArgs
+  ) => {
+    const story = await ctx.loaders.Stories.story.load(comment.storyID);
+    if (!story) {
+      throw new StoryNotFoundError(comment.storyID);
+    }
+    return retrieveCommentAllChildCommentsConnection(
+      ctx.mongo,
+      ctx.tenant.id,
+      comment,
+      {
+        first: 999,
```

Comment:
We may want to make this something bigger, like 9999 or even more, there are known streams of more than 1000 comments.

Reply:
I was thinking this would be fine for replies, but I agree that it could be good to bump it up to be safe 👍